sunpy/instr/lyra.py

```diff
@@ -586,7 +588,7 @@ def get_lytaf_event_types(lytaf_path=None, print_event_types=True):
                        for event_type in event_types]
     return all_event_types
 
-
+@deprecated("1.1")
```

Comment:
Side note, we have a known deprecation policy now, we could modify this decorator to say what version it will be removed in. Specifically I think it would be:

```python
major = int(v.split('.')[0]) + 1
minor = 1
removal_version = f"{major}.{minor}"
```

We should also then use this to have a way of listing all the functions that should be removed at the start of a release cycle...